### PR TITLE
add ci for vsphere multi-vcenter(upi)

### PR DESCRIPTION
**Pull request by WenXinWei**
this pr is aid to resolve bug :https://issues.redhat.com/browse/OCPBUGS-38635 and https://issues.redhat.com/browse/OCPBUGS-38604.

### CODE REVIEW

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-vsphere-upi-multi-vcenter-f28

**Comment by WenXinWei** [comment]
@jianlinliu @jinyunma please help review when you time time.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml [hunk 1]
```diff
@@ -3324,6 +3324,26 @@ tests:
...
+- as: vsphere-upi-multi-vcenter-f28
+  cron: 11 1 15 * *
+  steps:
+    cluster_profile: vsphere-elastic
+    env:
+      FEATURE_GATES: '["VSphereMultiVCenters=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
+    workflow: cucushift-installer-rehearse-vsphere-upi-zones
```
Comment (jinyunma) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:3336]: we don't need to set POOLs to make sure that two vcenters be used for this scenario as what we did for multi-vcenters IPI?

Comment (jinyunma) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:3336]: From job log you ran in this PR, I see cluster was deployed in single vcenter (vcenter.ci.ibmc.devcluster.openshift.com), multi datacenters, which is common zones scenario.

Comment (WenXinWei) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:3336]: thanks for reminding. i will update.

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.18-amd64-nightly-vsphere-upi-multi-vcenter-f28

**Comment by WenXinWei** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-vsphere-upi-multi-vcenter-f28

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/multi-vcenter/cucushift-installer-rehearse-vsphere-upi-multi-vcenter-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-vsphere-upi-multi-vcenter
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-vsphere-upi-multi-vcenter-provision
```
Comment (jinyunma) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/multi-vcenter/cucushift-installer-rehearse-vsphere-upi-multi-vcenter-workflow.yaml:5]: Seems content of this chain is same as `cucushift-installer-rehearse-vsphere-upi-zones-provision-chain`, I think we can directly use `cucushift-installer-rehearse-vsphere-upi-zones-provision-chain` here and also define the EVN `POOLS`, then new added chain can be removed.

**Comment by jinyunma** [comment]
@WenXinWei please also link the epic/story or bug with short explanation in PR description to reference in the future if needed.

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/multi-vcenter/cucushift-installer-rehearse-vsphere-upi-multi-vcenter-workflow.yaml (hunk 2 above)
Comment (WenXinWei) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/multi-vcenter/cucushift-installer-rehearse-vsphere-upi-multi-vcenter-workflow.yaml:5]: ok.

**Comment by jinyunma** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-vsphere-upi-multi-vcenter-f28

**Comment by jinyunma** [comment]
/lgtm

cc @liangxia to review

**Pull request merged**